next-sitemap.config.js

```diff
@@ -12,9 +12,20 @@ const sitemapConfig = {
   sitemapBaseFileName: 'sitemap-nb',
   changefreq: false,
   priority: false,
-
   generateRobotsTxt: false, // (optional)
-
+  transform: async (config, path) => {
```

Comment:
For posterity: https://github.com/iamvishnusankar/next-sitemap?tab=readme-ov-file#custom-transformation-function

Mind adding some inline comments here describing what we're doing and why we're doing it to make archeology easier in the future?